modules/artifact-activation/src/test/java/org/mule/runtime/module/artifact/activation/internal/deployable/LightweightDeployableProjectModelBuilderTestCase.java

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.runtime.module.artifact.activation.internal.deployable;
+
+import static org.mule.maven.client.api.MavenClientProvider.discoverProvider;
+import static org.mule.runtime.module.artifact.activation.internal.MavenTestUtils.installArtifact;
+import static org.mule.test.allure.AllureConstants.ClassloadingIsolationFeature.CLASSLOADING_ISOLATION;
+import static org.mule.test.allure.AllureConstants.ClassloadingIsolationFeature.ClassloadingIsolationStory.ARTIFACT_DESCRIPTORS;
+
+import static org.hamcrest.Matchers.contains;
+import static org.hamcrest.Matchers.equalTo;
+import static org.hamcrest.Matchers.hasProperty;
+import static org.junit.Assert.assertThat;
+
+import org.mule.runtime.module.artifact.activation.api.deployable.DeployableProjectModel;
+import org.mule.runtime.module.artifact.activation.internal.maven.LightweightDeployableProjectModelBuilder;
+import org.mule.tck.junit4.rule.SystemProperty;
+
+import java.io.File;
+import java.net.URISyntaxException;
+
+import io.qameta.allure.Feature;
+import io.qameta.allure.Story;
+import org.junit.Rule;
+import org.junit.Test;
+
+@Feature(CLASSLOADING_ISOLATION)
+@Story(ARTIFACT_DESCRIPTORS)
+public class LightweightDeployableProjectModelBuilderTestCase {
+
+  @Rule
+  public SystemProperty repositoryLocation = new SystemProperty("muleRuntimeConfig.maven.repositoryLocation",
```

Comment:
should this better use a temp file scoped to this test only instead of using the actual m2 repo from the machine?